compilers/4.03.0/4.03.0+32bit/4.03.0+32bit.descr

```diff
@@ -0,0 +1,2 @@
+Official 4.03.0 release
```

Comment:
Could you please change the description to something like the [description for `4.02.3+32bit`](https://github.com/ocaml/opam-repository/blob/48df3858c9c10aff9728083edee70660103ffde5/compilers/4.02.3/4.02.3%2B32bit/4.02.3%2B32bit.descr#L1):

```   OCaml 4.02.3 compiled in 32-bit mode for 64-bit Linux and OS X hosts```

?